policyengine_uk/variables/gov/hmrc/tax_free_childcare/tax_free_childcare.py

```diff
@@ -23,15 +23,23 @@ def formula(benunit, period, parameters):
         # Child gets higher amount if either disabled or blind
         qualifies_for_higher_amount = is_disabled | is_blind
 
-        amount_per_child = (
+        # Get childcare expenses
+        childcare_expense_per_child = benunit.members(
+            "childcare_expenses", period
+        )
+
+        # Calculate contribution using rate from parameters
+        contribution = childcare_expense_per_child * p.rate
```

Comment:
So given the above comment I think:

contribution / (contribution + childcare_expense) = rate
contribution = rate x (contribution + childcare_expense)
contribution x (1 - rate) = rate x childcare_expense
contribution = (rate x childcare_expense) / (1 - rate)